### PR TITLE
Implement firework methods

### PR DESCRIPTION
**Pull request by CaptainElliott**
Basic methods to implement fireworks into other modules

### CODE REVIEW

File: src/main/java/in/twizmwaz/cardinal/util/FireworkUtil.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    public static void spawnFirework(Location location, World world, Color color) {
+        Firework firework = (Firework) world.spawn(location, Firework.class);
+        FireworkMeta fireworkMeta = firework.getFireworkMeta();
+        FireworkEffect effect = FireworkEffect.builder().withColor(color).with(FireworkEffect.Type.BALL).trail(true).build();
+        fireworkMeta.addEffect(effect);
+        fireworkMeta.setPower(0);
+        firework.setFireworkMeta(fireworkMeta);
+    }
+
+    private static Color randomColor(int c) {
```
Comment (twizmwazin) [inline src/main/java/in/twizmwaz/cardinal/util/FireworkUtil.java]: Why the int if it is random? Random.nextInt(16) will return a pseudo-random int between 0-16.

**Comment by twizmwazin** [comment]
Other than that one method, this looks good.

**Comment by iEli2tyree011** [comment]
Firework#setPower(0) doesn't work as intended

**Comment by CaptainElliott** [comment]
The .setPower(0) was unintentional.

Also don't merge just yet

**Comment by iEli2tyree011** [comment]
What is the intent of setting the power of the firework?

**Comment by CaptainElliott** [comment]
@iEli2tyree011 Power is half a second of flight time.

@twizmwazin  This can now be merged

**Comment by twizmwazin** [comment]
Thanks

**Pull request merged**